app/jobs/dqt/sync_states_job.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Dqt
+  class SyncStatesJob < ApplicationJob
+    queue_as :dqt_sync
```

Comment:
Having these on the same queue as the `SyncTraineeStateJob` might cause them to be delayed if that queue starts to back up. I think if that happens it will be fine as that will mean we're not getting close to triggering the 300 request/minute rate limit in any case. We can just bear this in mind during the smaller runs.